<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>510</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>854</width>
    <height>510</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>906</width>
    <height>510</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MapleSeed++</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>861</width>
      <height>451</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="usesScrollButtons">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Main</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>10</y>
        <width>261</width>
        <height>371</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>561</width>
        <height>371</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>390</y>
        <width>831</width>
        <height>28</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QProgressBar {
border: 1px solid black;
text-align: center;
padding: 1px;
width: 15px;
}

QProgressBar::chunk {
background-color: #cd9bff;
border: 1px solid black;
}</string>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="textVisible">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="textDirection">
       <enum>QProgressBar::TopToBottom</enum>
      </property>
      <property name="format">
       <string>%p%</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Log</string>
     </attribute>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>851</width>
        <height>421</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>854</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuLog">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Log</string>
     </property>
     <addaction name="actionVerbose"/>
    </widget>
    <widget class="QMenu" name="menuConfig">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Config</string>
     </property>
     <addaction name="actionConfigTemporary"/>
     <addaction name="actionConfigPersistent"/>
    </widget>
    <addaction name="menuLog"/>
    <addaction name="menuConfig"/>
    <addaction name="actionClear_Settings"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuGame_Library">
    <property name="title">
     <string>Library</string>
    </property>
    <addaction name="actionChange_Library"/>
    <addaction name="actionRefreshLibrary"/>
    <addaction name="separator"/>
    <addaction name="actionOffline_Mode"/>
   </widget>
   <widget class="QMenu" name="menuContent">
    <property name="title">
     <string>Content</string>
    </property>
    <widget class="QMenu" name="menuDownload">
     <property name="title">
      <string>Download</string>
     </property>
     <addaction name="actionDownload_Title"/>
     <addaction name="actionUpdate"/>
     <addaction name="actionDLC"/>
    </widget>
    <addaction name="actionDecrypt_Content"/>
    <addaction name="menuDownload"/>
   </widget>
   <widget class="QMenu" name="menuCemu">
    <property name="title">
     <string>Cemu</string>
    </property>
    <widget class="QMenu" name="menuSave_Data">
     <property name="title">
      <string>Save Data</string>
     </property>
     <addaction name="actionBackup"/>
     <addaction name="actionImport"/>
    </widget>
    <addaction name="actionIntegrateCemu"/>
    <addaction name="menuSave_Data"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuGame_Library"/>
   <addaction name="menuContent"/>
   <addaction name="menuCemu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Exit the program</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionChange_Library">
   <property name="text">
    <string>Change</string>
   </property>
   <property name="toolTip">
    <string>Change the current games location</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionDownload_Title">
   <property name="text">
    <string>Title</string>
   </property>
  </action>
  <action name="actionDecrypt_Content">
   <property name="text">
    <string>Decrypt</string>
   </property>
  </action>
  <action name="actionVerbose">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Verbose</string>
   </property>
  </action>
  <action name="actionUpdate">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </action>
  <action name="actionDLC">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>DLC</string>
   </property>
  </action>
  <action name="actionBackup">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Backup</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionIntegrateCemu">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Integrate</string>
   </property>
  </action>
  <action name="actionConfigTemporary">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Temporary</string>
   </property>
  </action>
  <action name="actionConfigPersistent">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Persistent</string>
   </property>
  </action>
  <action name="actionRefreshLibrary">
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionOffline_Mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Offline Mode</string>
   </property>
  </action>
  <action name="actionClear_Settings">
   <property name="text">
    <string>Clear Settings</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
